packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -336,7 +336,7 @@ export class DetailsRowBase extends BaseComponent<IDetailsRowBaseProps, IDetails
   }
 
   protected _onRenderCheck(props: IDetailsRowCheckProps) {
-    return <DetailsRowCheck {...props} />;
+    return <DetailsRowCheck {...props} aria-label={props.title} />;
```

Comment:
Instead, you could probably add

```
"aria-label": checkButtonAriaLabel,
```

to the `onRenderCheck` method call on line 271

Reply:
Updated.